src/osd/PGLog.h

```diff
@@ -1257,6 +1257,7 @@ struct PGLog : DoutPrefixProvider {
     bool maintain_rollback,
     IndexedLog *log,
     missing_type &missing,
+    bool &should_rollforward,
```

Comment:
Out params like should_rollforward should be passed as a pointer with null allowed as a way for the caller to ignore the output, see CodingStyle.

Reply:
thanks, will take into accord.
updated code.